lib/ckb/address.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module CKB
+  class Address
+    attr_reader :pubkey
+
+    MODE_TESTNET = "testnet"
+    MODE_MAINNET = "mainnet"
+    MODE_CUSTOM = "custom"
+
+    PREFIX_MAINNET = "ckb"
+    PREFIX_TESTNET = "ckt"
+
+    def initialize(pubkey, mode: MODE_TESTNET)
+      @pubkey = pubkey
+      @prefix = if mode == MODE_TESTNET
+                  PREFIX_TESTNET
+                elsif mode == MODE_MAINNET
+                  PREFIX_MAINNET
+                end
+    end
+
+    def blake160
+      @blake160 ||= self.class.blake160(@pubkey)
+    end
+
+    # Generates address assuming default lock script is used
+    # payload = type(01) | bin-idx("P2PH" => "50/32/50/48") | pubkey blake160
+    # see https://github.com/nervosnetwork/ckb/wiki/Common-Address-Format for more info.
+    def generate
+      blake160_bin = [blake160[2..-1]].pack("H*")
+      payload = ["0150325048"].pack("H*") + blake160_bin
```

Comment:
`0150325048` is too magic even with the payload comment. 🤔

Reply:
How about this https://github.com/nervosnetwork/ckb-sdk-ruby/pull/26/files#diff-d1fae9cd48181daa7305586f3950fc2fR29